dbt/models/eia923/out_eia923__monthly_generation_fuel_combined/schema.yml

```diff
@@ -32,4 +32,29 @@ sources:
           - name: fuel_consumed_mmbtu
           - name: fuel_consumed_for_electricity_mmbtu
           - name: net_generation_mwh
+            data_tests:
+              - expect_column_fraction_with_condition:
+                  numerator_row_condition: fuel_type_code_pudl='nuclear'
+                  min_fraction: 0.18
+                  max_fraction: 0.22
+              - expect_column_fraction_with_condition:
+                  numerator_row_condition: fuel_type_code_pudl='coal'
+                  min_fraction: 0.18
+                  max_fraction: 0.22
+              - expect_column_fraction_with_condition:
+                  numerator_row_condition: fuel_type_code_pudl='gas'
+                  min_fraction: 0.30
+                  max_fraction: 0.50
+              - expect_column_fraction_with_condition:
+                  numerator_row_condition: fuel_type_code_pudl='hydro'
+                  min_fraction: 0.04
+                  max_fraction: 0.10
+              - expect_column_fraction_with_condition:
+                  numerator_row_condition: fuel_type_code_pudl='wind'
+                  min_fraction: 0.05
+                  max_fraction: 0.15
+              - expect_column_fraction_with_condition:
+                  numerator_row_condition: fuel_type_code_pudl='solar'
+                  min_fraction: 0.00
+                  max_fraction: 0.10
```

Comment:
Tried applying this test more broadly for other generation types. It passes locally, we'll see if it needs to be tweaked to pass CI / Fast ETL.